projects/vc-api-implementation.md

```diff
@@ -0,0 +1,226 @@
+# Project Name
+
+VC API Implementation
+
+# Project Description
+
+The VC API is specification for a set of APIs for VC lifecycle management (https://w3c-ccg.github.io/vc-api/). This includes operations such as credential issuance, verification and exchange. It is a W3C CCG work item and, as of the submission of this proposal, it is in “draft community report” status. 
+
+The VC API’s design is informed by use cases across a range of domains. Several of these use cases are collected in a working group note (https://w3c-ccg.github.io/vc-api-use-cases/). 
+
+This project is an implementation of the VC API. The implementation aims to enable organizations and individuals to effortlessly conduct SSI operations over HTTP without requiring technical expertise, making it seamless to integrate into existing projects. 
+
+This implementation does not yet implement all of the methods of VC API.
+The list of API methods which are implemented is in a table in the [application README](https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api#implemented-endpoints).
```

Comment:
@troyronda to this question:
> Is this project based on the latest version of the CCG VC API? Does it already implement the full community report?

I have added a link to this list of VC API methods implemented by this project. The project is currently a partial implementation of the specification.